plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -166,20 +176,37 @@ func PrioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		priorityFunc := priorityConfig.Function
 		prioritizedList, err := priorityFunc(pod, podLister, minionLister)
 		if err != nil {
-			return algorithm.HostPriorityList{}, err
+			return schedulerapi.HostPriorityList{}, err
 		}
 		for _, hostEntry := range prioritizedList {
 			combinedScores[hostEntry.Host] += hostEntry.Score * weight
 		}
 	}
+	if len(extenders) != 0 && minionLister != nil {
+		nodes, err := minionLister.List()
+		if err != nil {
+			return schedulerapi.HostPriorityList{}, err
+		}
+
+		for _, extender := range extenders {
+			prioritizedList, err := extender.Prioritize(pod, &nodes)
```

Comment:
@davidopp Thanks for the review.

The concern here is that the pod list can be too large. The other assumption I made is that any inter pod synergies for scheduling purpose can be handled by the k8s scheduler (affinity, anti affinity etc).